code/modules/power/smes.dm

```diff
@@ -14,9 +14,12 @@
 #define SMES_INPUTTING 8
 #define SMES_INPUT_ATTEMPT 9
 
+#define SMESEMPTIME 20 SECONDS // the time it takes for the SMES to go back to normal operation when emped
```

Comment:
why is this a define? it's only used once?
Also, if it remains one please format it consistently with the others